src/main/java/com/checkout/apm/ideal/IdealInfo.java

```diff
@@ -1,13 +1,16 @@
 package com.checkout.apm.ideal;
 
+import com.checkout.HttpMetadata;
 import com.checkout.common.Link;
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 
 import java.util.List;
 
+@EqualsAndHashCode(callSuper = true)
 @Data
```

Comment:
Can you please put `@EqualsAndHashCode` below `@Data`? I think it improves readability (is a data class before it overrides equals and hascode)